CS3219-AY2324S1/ay2324s1-course-assessment-g33
# System testing

# System Testing with Selenium and CI/CD

## Description
This pull request introduces system tests using Selenium for the following functionalities:
- Login
- Logout
- Signup Page
- Problems Page
- Dashboard Page
- Routing

Updated GitHub Actions Workflow YAML:
The .github/workflows/system_test.yml file has been updated to include the configuration for running system tests. 

## How to Test
1. Install the necessary dependencies using `pip install -r requirements.txt`.
2. Ensure that the Selenium WebDriver is correctly configured.
3. Run the system tests using `pytest` in the system_test folder.
